tensorflow/lite/micro/kernels/add_n_test.cc

```diff
@@ -28,7 +28,7 @@ namespace {
 constexpr int kMaxInputTensors = 3;
 constexpr int kMaxOutputTensors = 1;
 
-void ExecuteAddN(TfLiteTensor* tensors, int tensors_count) {
+void ExecuteAddN(TfLiteTensor* tensors, const int tensors_count) {
```

Comment:
Could you leave this as-is, unless there's a reason this is necessary? Even though it's cleaner your way, it's nice to keep the code churn to a minimum.

Reply:
Fixed.